nucypher/cli/characters/ursula.py

```diff
@@ -358,28 +366,43 @@ def ursula(click_config,
 
         if not force:  # Review
             click.echo("""
+| Staged Stake |
+|{address}|
 
-            | Staged Stake |
-
-            Node: {address}
-            Value: {value}
-            Duration: {duration}
-            Start Period: {start_period}
-            End Period: {end_period}
+{nu_value} NU | ({wei_value} NU-wei)
+{duration} Periods
 
+Start Period: {start_period}
+Expiration: {end_period}
             """.format(address=URSULA.checksum_public_address,
-                       value=value,
+                       wei_value=stake_wei,
+                       nu_value=nu_value,
                        duration=duration,
                        start_period=start_period,
                        end_period=end_period))
 
-            click.confirm("Publish Stake?", abort=True)
+            click.confirm(f"""
+By agreeing to lock {nu_value} NU, you are also obligated to 
+maintain a networked and available Ursula with the ETH address {URSULA.checksum_public_address} 
+for the duration of the stake(s) ({duration} periods), and allow the
+NuCypher network users to carry-out uninterrupted re-encryption 
+work orders without interference. 
+ 
+Failure to keep your node online or violation or re-encryption work orders
```

Comment:
My only qualm in this PR is this line. It's only true for invalid re-encryptions.

Reply:
Yeah - It's not perfect language.  What can be done to improve it?

Reply:
I'll leave it for a Discord conversation, no worries.